scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,442 @@
+aqueous protons. This exchange attenuates the MRI signal, introducing
+microstructure-dependent contrast. MT's effect reflects the relative density
+of macromolecules such as proteins and lipids, it has been associated with
+myelin content in white matter of the brain.
+
+Different contrasts can be done with an off-resonance pulse prior to image
+acquisition (a prepulse), saturating the protons on non-aqueous molecules,
+by applying different frequency irradiation. The two MT maps and two ihMT maps
+are obtained using five contrasts: single frequency positive or negative and
+dual frequency with an alternation of both positive and negative frequency
+(saturated images); and one unsaturated contrast as reference (T1weighted).
+
+The output consist in two types of images:
+        five contrast images and four (ih)MT maps.
+
+Contrast images :
+    - AltNP : alternating negative and positive frequency pulses
+    - altPN : alternating positive and negative frequency pulses
+    - positive : pulses applied at positive frequency
+    - negative : pulses applied at negative frequency
+    - reference : no pulse
+
+Maps :
+    Magnetization Transfer (MTR) and inhomogeneous (ihMTR) Ratio maps
+          The (ih)MT ratio is a measure reflecting the amount of bound protons.
+    Magnetization Transfer (MTsat) and inhomogeneous (ihMTsat) saturation maps
+          The (ih)MT saturation is a pseudo-quantitative ihMT maps representing
+          the signal change between the bound and free water pools.
+
+"""
+
+import argparse
+import os
+
+import json
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('id_subj',
+                   help='Name of Subject for saving output files.')
+    p.add_argument('out_dir',
+                   help='Path to output folder.')
+    p.add_argument('in_mask',
+                   help='Path to the T1 binary brain mask. Must be the sum '
+                        'of the three tissue probability maps from '
+                        'T1 segmentation (GM+WM+CSF).')
+    p.add_argument('--filtering', action='store_true',
+                   help='Gaussian filtering to remove Gibbs ringing.'
+                        'Not recommanded.')
+
+    g = p.add_argument_group(title='ihMT contrasts', description='Path to '
+                             'echoes corresponding to contrasts images. All '
+                             'constrat must have the same number of echoes '
+                             'and coregistered between them.'
+                             'Use * to include all echoes.')
```

Comment:
Ok here is the information I was requesting before. Maybe put that info above as well.

Also, missing space before 'Use'